inc/zoo/swar/SWAR.h

```diff
@@ -107,10 +107,12 @@ constexpr uint64_t popcount(uint64_t a) noexcept {
 }
 
 
+/// Index into the bits of the type T that contains the MSB.
 template<typename T> constexpr typename std::make_unsigned<T>::type msb_index(T v) noexcept {
     return 8*sizeof(T) - 1 - __builtin_clzll(v);
 }
 
+/// Index into the bits of the type T that contains the LSB.
 template<typename T> constexpr typename std::make_unsigned<T>::type lsb_index(T v) noexcept {
```

Comment:
why the `snake_case` instead of the `bactrianCase`?

Reply:
It was there before, fixing.